### PR TITLE
feat(order-details): feature/icky 627view order details

### PR DESCRIPTION
**Pull request by TriptiVermaXP**

### CODE REVIEW

File: components/order/view-order-details/ViewOrderDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+            <Box sx={{ ...styles.heading }}>
+              <Typography variant="h4" fontWeight={700}>
+                {t('pickup')}
+              </Typography>
+              <Typography variant="h4" fontWeight={700} color="primary">
+                {t('est-pickup')} {orderGetters.getExpectedDeliveryDate(pickupItems)}
+              </Typography>
+            </Box>
+            <ProductItemList items={pickupItems} />
+            <AddressCard {...(payments && payments[0]?.billingInfo?.billingContact?.address)} />
```
Comment (kibo-kevinwatts) [inline components/order/view-order-details/ViewOrderDetails.tsx:119]: go ahead and add a prop for this, `storePickupAddress?: Address` to use here

File: components/common/ProductItem/ProductItem.spec.tsx [hunk 2]
```diff
@@ -19,39 +20,57 @@ const onClickStoreLocatorMock = jest.fn()
...
   it('should call onClickStoreLocatorMock when click onClickStoreLocator', () => {
     render(
       <WithChageStoreOption
         {...WithChageStoreOption.args}
+        isPickupItem={true}
         onClickStoreLocator={onClickStoreLocatorMock}
       />
     )
-    const changeStore = screen.getByTestId('change-store-MS-BTL-002')
+    const changeStore = screen.getByTestId('change-store-MS-BTL-004')
```
Comment (kibo-kevinwatts) [inline components/common/ProductItem/ProductItem.spec.tsx:73]: this test id looks dynamic and based on the mock data
instead, can you get by text?

**Review by kibo-kevinwatts — COMMENTED** [review]
Nice work!

File: components/order/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export { default as ShippingStep } from './view-order-details/ViewOrderDetails'
```
Comment (kibo-kevinwatts) [inline components/order/index.ts:1]: I think "ShippingStep" should be "ViewOrderDetails" or something order details related :)

File: components/order/view-order-details/ViewOrderDetails.tsx (hunk 1 above)
Comment (TriptiVermaXP) [reply components/order/view-order-details/ViewOrderDetails.tsx]: Done

File: components/order/index.ts (hunk 3 above)
Comment (TriptiVermaXP) [reply components/order/index.ts]: Done

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**